openshift/release/update-to-head.sh

```diff
@@ -39,6 +39,11 @@ git checkout upstream/main -B release-next
 
 # Update openshift's main and take all needed files from there.
 git fetch openshift main
+
+git show openshift/main:patches/internal-registry.patch > internal-registry.patch
+git am internal-registry.patch
+rm -fr internal-registry.patch
+
```

Comment:
I wonder whether it would be more future proof to already checkout the whole patch directory and apply all patches within this directory? That way in the future you only have to add a new patch to this directory but do not have to touch this script anymore.

Reply:
When using `git checkout` it adds files to staging which breaks `git am`. I don't know how to prevent staging.

Reply:
@rhuss does it make sense?